Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:10.4.1
+
+COPY . /app
+WORKDIR /app
+RUN npm install -g nodemon
```

Comment:
Does installing `nodemon` really make sense here? You've hard-copied the files into the Docker image, thus they'll never change.